package.json

```diff
@@ -29,10 +29,10 @@
 		"reflect-metadata": "^0.1.13",
 		"retire": "^2.2.5",
 		"semver": "^7.3.4",
-		"ts-node": "^8",
+		"ts-node": "^10",
 		"tslib": "^2",
 		"tsyringe": "^4.1.0",
-		"typescript": "^3.8.2",
+		"typescript": "^4.6.2",
```

Comment:
In addition to updating `typescript`, I also had to update `@salesforce/command` and `ts-node`, and push an updated `yarn.lock` file. Otherwise, the `npm pack` step in CI's `setup` job would fail.

Reply:
What are your thoughts on deleting `yarn.lock` altogether since our users don't get it anyway? Though there could be some value since it makes sure everyone in the team is using the same version of dependencies.

Reply:
I had to update the `yarn.lock` file in order for the `npm pack` step that the CI runs to succeed. I like the idea of removing `yarn.lock` in the long-term, but I'm a bit nervous about it in the short-term.